### PR TITLE
Support Python 3

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This introduces support for Python 3 in a manner which is still compatible with Python 2.

The latter is needed as the SR kits are (for the moment) still Python 2.

Note: This currently depends on a change I've lined up to `PyPyBox2D`: https://github.com/pybox2d/pypybox2d/pull/3, though I've not expressed that anywhere in the dependencies. I'm hoping that we'll get a response to that PR so the fix will be on PyPI.

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
ping @Adimote for review
cc @HarryCutts for info

File: sr/robot/sim_robot.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from __future__ import division
```
Comment (Adimote) [inline sr/robot/sim_robot.py:3]: is this needed any more?

**Review by Adimote — APPROVED** [review]
LGTM but you might have missed a `from __future__`?

File: sr/robot/sim_robot.py (hunk 1 above)
Comment (PeterJCLaw) [reply sr/robot/sim_robot.py:3]: Yes. The intent here is to add support for Python 3, not to move entirely to it.
These imports are needed for Python 2 (see also the commit message of https://github.com/srobo/sr-turtle/pull/3/commits/8a364cb0efdbb5f3b8c2789e1b618e3db2cef7a8).

**Comment by HarryCutts** [comment]
Looks good to me, thanks Peter!

**Pull request merged**